Quick Enroll on a Ubuntu 18.04 node does not result in enroll.sh download and install.

Hi Javier,
    I have deployed osctrl using, make docker_all. I have created a new environment (and copied the certificates of the dev environment) using the admin interface. I was attempting to enroll an Ubuntu 18.04.3 LTS using the quick-enroll method. On running the enrollment link using curl, agent (enroll.sh) does not download and install. I have attached the logs generated by each container as well as the curl on the remote machine. I would like to add that the system is on the local network and accessible as the logs show. Please help me with this.
[osctrl-enroll-file-download-fail-issue.txt](https://github.com/jmpsec/osctrl/files/4627055/osctrl-enroll-file-download-fail-issue.txt)

Those logs indicate that the service `osctrl-tls` has not been deployed correctly since it is missing some files:

```
osctrl-tls          | handlers.go:537: error getting script open scripts/quick-add.sh: no such file or directory
```

I fixed the above issue, using this fix. https://github.com/openssl/openssl/issues/7754#issuecomment-586713820

Now i dont see the error in the logs.
[+] Preparing certificates for osctrl-nginx
[+] Generating deploy/docker/certs/osctrl.key and deploy/docker/certs/osctrl.crt
Generating a RSA private key
........................................+++++
..........................................................+++++
writing new private key to 'deploy/docker/certs/osctrl.key'

Unfortunately, i still see the errors 
osctrl-tls          | handlers.go:537: error getting script open scripts/quick-add.sh: no such file or directory

and the enroll.sh script does not download and install. I am now testing this on a physical machine. Osquery is installed and running on the machine.

--- a/deploy/docker/dockerize.sh
+++ b/deploy/docker/dockerize.sh
@@ -78,7 +78,7 @@ mkdir -p "$CERTSDIR"
 mkdir -p "$CONFIGDIR"
 
 # Secret for API JWT
-_JWT_SECRET="$(head -c64 < /dev/random | base64 | head -n 1 | openssl dgst -sha256 | cut -d " " -f1)"
+_JWT_SECRET="$(head -c64 < /dev/random | base64 | head -n 1 | openssl dgst -sha256 | cut -d " " -f2)"
 
 # Default values for arguments
 SHOW_USAGE=true

--- a/deploy/docker/tls/Dockerfile
+++ b/deploy/docker/tls/Dockerfile
@@ -10,6 +10,8 @@ WORKDIR /osctrl-tls
 
 COPY . .
 
+COPY tls/scripts/ scripts
+
 RUN go build -o bin/osctrl-tls tls/*.go
 RUN go build -o bin/osctrl-cli cli/*.go
 

